src/runner/index.ts

```diff
@@ -35,6 +35,17 @@ export default async function run(
   const oldCommand: string | undefined =
     pkg && pkg.scripts && pkg.scripts[hookName.replace('-', '')]
 
+  const activeBranches: string | undefined =
```

Comment:
Maybe it can be an array of strings. What do you think?

Reply:
It should indeed be an array, that code would result in an error. Thanks for pointing it out!